libs/libarchfpga/src/histogram.cpp

```diff
@@ -9,17 +9,16 @@
 
 #include "histogram.h"
 
-std::vector<HistogramBucket> build_histogram(std::vector<float> values, size_t num_bins, float min_value, float max_value) {
+std::vector<HistogramBucket> build_histogram(std::vector<float> values,
+                                             size_t num_bins,
+                                             float min_value,
+                                             float max_value) {
     std::vector<HistogramBucket> histogram;
 
     if (values.empty()) return histogram;
 
-    if (std::isnan(min_value)) {
-        min_value = *std::min_element(values.begin(), values.end());
-    }
-    if (std::isnan(max_value)) {
-        max_value = *std::max_element(values.begin(), values.end());
-    }
+    if (std::isnan(min_value)) { min_value = *std::min_element(values.begin(), values.end()); }
+    if (std::isnan(max_value)) { max_value = *std::max_element(values.begin(), values.end()); }
```

Comment:
AllowShortIfStatementsOnASingleLine: true

Reply:
Turn off ! (Ugh.)

Reply:
I agree with Vaughn. I think it should not be required. Its nice if it was optional, for certain places, but I do not want to make this required. The way Clang Format is set up it will automatically set it to this version of the line is small enough.